modulefiles/module_base.hera.lua

```diff
@@ -41,11 +41,10 @@ load(pathJoin("py-xarray", (os.getenv("py_xarray_ver") or "None")))
 setenv("WGRIB2","wgrib2")
 setenv("UTILROOT",(os.getenv("prod_util_ROOT") or "None"))
 
---prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/prepobs/v" .. (os.getenv("prepobs_run_ver") or "None"), "modulefiles"))
-prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/prepobs/feature-GFSv17_com_reorg_log_update/modulefiles"))
+prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/prepobs/feature-hera-rocky8/modulefiles"))
 load(pathJoin("prepobs", (os.getenv("prepobs_run_ver") or "None")))
 
-prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/Fit2Obs/v" .. (os.getenv("fit2obs_ver") or "None"), "modulefiles"))
+prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/global/glopara/git/Fit2Obs/feature-hera-rocky8/modulefiles"))
```

Comment:
Alrighty, so we merged the Fit2Obs PR with the Rocky8 changes for Hera, I cut a version tag, and installed it here on Hera: `/scratch1/NCEPDEV/global/glopara/git/Fit2Obs/v1.1.0`.

Please revert this back to the prior line that includes `fit2obs_ver` and then update `fit2obs_ver` in `run.spack.ver` to be `export fit2obs_ver=1.1.0`. I will install this new tag on our platforms on Monday.
https://github.com/NOAA-EMC/global-workflow/blob/develop/versions/run.spack.ver#L34

Reply:
Looks good, thanks!